components/feature/addons/src/main/java/mozilla/components/feature/addons/update/AddonUpdater.kt

```diff
@@ -269,7 +270,7 @@ class DefaultAddonUpdater(
         extension: WebExtension,
         newPermissions: List<String>
     ): Notification {
-        val notificationId = SharedIdsHelper.getIdForTag(applicationContext, NOTIFICATION_TAG)
+        val notificationId = NotificationHandlerService.getNotificationId(applicationContext, extension.id)
```

Comment:
The main issue that prevented multiple notifications from showing was that we were using the same notification id, now to create the notification id we are including the `extension.id` as well, making the notification unique per extensions.